packages/hash/frontend/theme-override.d.ts

```diff
@@ -0,0 +1,98 @@
+declare module "@mui/material/styles" {
+  interface Theme {
+    borderRadii: {
+      none: string;
+      sm: string;
+      md: string;
+      lg: string;
+    };
+  }
+
+  interface ThemeOptions {
+    borderRadii?: {
+      none?: string;
+      sm?: string;
+      md?: string;
+      lg?: string;
+    };
```

Comment:
Why are all these optional? If we're adding them to the theme, can't we declare that they'll always be accessible?
```suggestion
    borderRadii: {
      none: string;
      sm: string;
      md: string;
      lg: string;
    };
```

Reply:
Fixed in 877d708